test/functional/basic/gesture-e2e-specs.js

```diff
@@ -223,6 +223,29 @@ describe('XCUITestDriver - gestures', function () {
         await doPinch();
       });
     });
+    describe('special actions', function () {
+      it('should open the control center by swiping up at the bottom', async function () {
+        await driver.elementByAccessibilityId('ControlCenterView')
```

Comment:
is `ControlCenterView` something that xcuitest actually returns as the element type, or is it a special thing we made?

Reply:
In my experiments, that is what XCUITest returns for the thingo down there.